Package@swift-4.swift

```diff
@@ -0,0 +1,44 @@
+// swift-tools-version:4.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+/**
+ * Copyright IBM Corporation 2016, 2018
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import PackageDescription
+
+let package = Package(
+    name: "KituraStencil",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "KituraStencil",
+            targets: ["KituraStencil"]
+        )
+    ],
+    dependencies: [
+        .package(url: "https://github.com/IBM-Swift/Kitura-TemplateEngine.git", .upToNextMinor(from: "1.7.3")),
```

Comment:
Why are we being very restrictive with the version?

Reply:
1.7.3 is the minimum version that will build with Swift 4.1.  Previously the Swift 4 Package.swift was named `Package@swift-4.0.swift`, which is too specific about the Swift version, and hence ignored by Swift 4.1.  1.7.3 renames this to `Package@swift-4.swift`.
See: https://github.com/IBM-Swift/Kitura-TemplateEngine/compare/1.7.2...1.7.3